packages/calling-component-bindings/src/providers/CallProvider.tsx

```diff
@@ -66,7 +66,7 @@ export const useCall = (): Call | undefined => {
   if (call && !_isACSCall(call)) {
     throw new Error('Incorrect call type: Must provide a Regular Call object.');
   }
-  return call;
+  return call as Call;
```

Comment:
@PorterNan  do we need to do this "cast"?

Reply:
we shouldn't need to.

Reply:
Not needed, just added to avoid warnings in VSCode I can remove if necessary